examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stationarity in time-series\n",
+    "\n",
+    "Stationarity means that the statistical properties of a a time series i.e. mean, variance and covariance do not change over time. Most statistical models require the series to be stationary to make effective and precise predictions.\n",
+    "\n",
+    "We will implement two statistical tests to check the stationarity of a time series – Augmented Dickey Fuller (“ADF”) test and Kwiatkowski-Phillips-Schmidt-Shin (“KPSS”) test. We shall also implement a method to convert a non-stationary time series into stationary. "
```

Comment:
Instead of "implement two statistical tests", it would be better to say "use two statistical tests", since this notebook uses the tests that are already implemented in Statsmodels.

Reply:
ok, I shall correct it.